code/mob/living/life/Life.dm

```diff
@@ -693,6 +693,9 @@
 		if (bioHolder?.HasEffect("food_rad_resist"))
 			rad_protection += 100
 
+		if (src.hasStatus("food_rad_resist"))
```

Comment:
uhh whats the above thing

Reply:
the genetic rad immunity effect, which has the exact same ID.

Reply:
My theory as to why is that rad-wick came first, with 50% resistance, rad resist gene came along with the exact same effect, under some system that included both, where it eventually got buffed to 100%, while rad-wick got left behind, broken.